web3rpc/rpc-specs/paths/klay/miscellaneous/getBlsInfos.yaml

```
@@ -0,0 +1,105 @@
+openapi: "3.0.2"
+info:
+  title: Klay Open API
+  version: "1.10.0"
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+
+servers:
+  - url: http://localhost:7151
+
+paths:
+  /klay/miscellaneous/getBlsInfos:
+    post:
+      summary: "[Miscellaneous] klay_getBlsInfos"
+      tags:
+        - klay
+      operationId: getBlsInfos
+      description: |
+        Returns the account and storage values of the specified account including the Merkle-proof. This call can be used to verify that the data you are pulling from is not tampered with.
```

You can find the description in the https://kips.klaytn.foundation/KIPs/kip-113 document. Please copy the description, return value fields, and examples from there.
```suggestion
        Returns all registered BLS public keys and proof-of-possessions of the validators.
```